.github/workflows/genesis_files_for_relay.yml

```diff
@@ -0,0 +1,76 @@
+name: create relay genesis files
+on:
+  workflow_dispatch:
+    inputs:
+      chain:
+        description: The name of the chain to generate files for
+        required: true
+      version:
+        description: A **released** version of manta to generate the genesis files from in the form vX.Y.Z
+        required: true
+env:
+  CHAIN: ${{github.event.inputs.chain}}
+  VERSION: ${{github.event.inputs.version}}
+jobs:
+  export_genesis_files:
+    runs-on: ubuntu-latest
+    steps:
+      - name: get curl
+        run: |
+          apt update
+          apt install -y curl
+      - name: fetch and chmod manta release
+        run: |
+          curl -L -o $HOME/manta https://github.com/Manta-Network/Manta/releases/download/$VERSION/manta
+          chmod +x $HOME/manta
+          ls -ahl $HOME
+      # generate genesis files
+      - name: print genesis hash
+        run: |
+          $HOME/manta build-spec --chain $CHAIN --raw > ./$CHAIN-genesis.json
+          sha256sum $CHAIN-genesis.json | tee $CHAIN-genesis.json.sha256
+      - name: Export genesis state
+        run: |
+          $HOME/manta export-genesis-state --chain $CHAIN > $CHAIN-genesis.state
+          sha256sum $CHAIN-genesis.state | tee $CHAIN-genesis.state.sha256
+      - name: Export genesis wasm
+        run: |
+          $HOME/manta export-genesis-state --chain $CHAIN > $CHAIN-genesis.wasm
```

Comment:
export wasm not state here, also I've always used --raw

Reply:
thx, fixed the copypasta

according to https://docs.substrate.io/tutorials/connect-relay-and-parachains/acquire-a-testnet-slot/#export-required-files
these don't need to be `--raw`. I'd keep em that way, they did get registered successfully by shumo the last time around without this flag too

Reply:
Ok lemme just double check this doc, i'm not even sure what the flag did.

Reply:
So the output is going to be hex instead of binary, so that should be fine.